examples/autocomplete/package.json

```diff
@@ -19,7 +19,7 @@
     "react": "16.8.6",
     "react-autosuggest": "9.4.3",
     "react-dom": "16.8.6",
-    "react-instantsearch-dom": "5.7.0"
```

Comment:
this means the example no longer works on codesandbox and I use this quite frequently; can we make this change only for the ecommerce demo?

Reply:
We updated all examples on `instantsearch.js` so I made the same here.

But I see no issue to only do it on the e-commerce demo if it's better for you.

Reply:
yes, especially the server side rendering ones for example these are used for, thanks!

Reply:
Done, but I did not touch the other "real" demos (default-theme, media, tourism) since they are all using the same webpack configuration as e-commerce. Is it ok for you?